spec/unit/outputs/elasticsearch_spec.rb

```diff
@@ -1140,7 +1160,7 @@
   describe "post-register ES setup" do
     let(:do_register) { false }
     let(:es_version) { '7.10.0' } # DS default on LS 8.x
-    let(:options) { { 'hosts' => '127.0.0.1:9999' } }
+    let(:options) { { 'hosts' => '127.0.0.1:9999', 'data_stream' => 'false' } }
```

Comment:
```suggestion
    let(:options) { { 'hosts' => '127.0.0.1:9999', 'data_stream' => false } }
```

Reply:
For some reason, several tests fail on this spec if I set it as a boolean instead of a string! It looks the other tests are doing the same! 🤷

Reply:
Uhhh, ok would be nice to understand why, but however is out of context of this PR

Reply:
the setting is a string that can have values of 'true', 'false', and 'auto', so it makes sense to pass a string here too.